cmake/thirdparty/SetupCUDA.cmake

```diff
@@ -7,6 +7,15 @@
 # Sanity Checks
 ################################
 
+# Ensure CUDA_TOOLKIT_ROOT_DIR is specified, since it is needed 
+# by the call to find_packag(CUDA) below.
+if (NOT DEFINED CUDA_TOOLKIT_ROOT_DIR)
+   message( FATAL_ERROR
+        "Please specifiy CUDA_TOOLKIT_ROOT_DIR to use CUDA" )
```

Comment:
Typo: `specifiy` -> `specify`

Reply:
Done - Resolve and amended previous commit.